sci-geosciences/gmt/gmt-5.4.4.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit bash-completion-r1 cmake-utils
+
+DESCRIPTION="Powerful map generator"
+HOMEPAGE="https://gmt.soest.hawaii.edu/"
+SRC_URI="mirror://gmt/${P}-src.tar.xz"
+
+LICENSE="GPL-2 gmttria? ( Artistic )"
+SLOT="5"
+KEYWORDS="~amd64 ~x86"
+IUSE="+dcw doc examples +fftw +gdal gmttria +gshhg htmldoc lapack openmp pcre pcre2 threads tutorial"
+
+RDEPEND="
+	!sci-biology/probcons
+	app-text/ghostscript-gpl
+	dcw? ( sci-geosciences/dcw-gmt-data )
+	fftw? ( sci-libs/fftw:3.0/3 )
+	gdal? ( sci-libs/gdal )
+	gshhg? ( sci-geosciences/gshhg-data )
+	>=sci-libs/netcdf-4.1[hdf5]
+	lapack? ( virtual/lapack )
+	pcre? ( dev-libs/libpcre )
+	pcre2? ( dev-libs/libpcre2 )
+	net-misc/curl
+	sys-libs/zlib"
+
+DEPEND="${RDEPEND}"
+
+REQUIRED_USE="
+	pcre? ( !pcre2 )
+	pcre2? ( !pcre )"
+
+src_prepare() {
+	cmake-utils_src_prepare
+	# Rename man pages to avoid a name conflict with gmt4
+	pushd man_release || die
+	local m c suffix newc
+	for m in *.gz; do
+		c=${m%%.*}
+		suffix=${m#*.}
+		# This man pages does'nt conflict
+		if [ "$c" == "gmt" ]; then continue; fi
```

Comment:
Always use `[[ ... ]]` and `${foo}` forms. Also, this looks like a case for `case`.

Reply:
Yes, done